com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -164,6 +164,7 @@ public UniversalRenderPipeline(UniversalRenderPipelineAsset asset)
         {
 #if UNITY_EDITOR
             m_GlobalSettings = UniversalRenderPipelineGlobalSettings.Ensure();
+            ReloadResourcesIfNeeded();
```

Comment:
I wonder if it makes most sense and is less risky, to put this inside the Ensure of GlobalSettings. I think in the future it might even be nicer to move some of default settings to global settings instead of the pipeline asset?

Reply:
Currently, each URPAsset runs the reload separately.
But if moved to GlobalSettings we should iterate over each URPAsset in the project.

Reply:
ah ok, makes sense!